manage.py

```diff
@@ -35,5 +35,24 @@ def test():
     unittest.TextTestRunner(verbosity=2).run(tests)
 
 
+@manager.command
+def setup_dev():
```

Comment:
I think we've been treating `db.drop_all()` too lightly. I think we should separate out the drop and create commands into a separate manager command (perhaps named `recreate_db`). This would mean that `setup_dev()` and `setup_prod()` would be the same, but I think that's okay for now. Open to other solutions, I just don't feel comfortable with destroying the database in a command called `setup_dev()`.